README.md

```diff
@@ -0,0 +1,77 @@
+# flatcar-packer-qemu
+Packer QEMU image builder for Flatcar Linux
+
+A basic Packer template, Makefile and cloud config file to get started building QEMU images for [Flatcar Linux](https://flatcar-linux.org/).
```

Comment:
Guess this can be merged with the sentence above.

Also, `s/cloud config/ignition config/`